extensions/default/src/utils/validations/areAllImagePositionsEqual.ts

```diff
@@ -1,7 +1,7 @@
 import { vec3 } from 'gl-matrix';
 import toNumber from '@ohif/core/src/utils/toNumber';
 import { _getPerpendicularDistance } from '@ohif/core/src/utils/isDisplaySetReconstructable';
-import calculateScanAxisNormal from '../calculateScanAxisNormal';
+import calculateScanAxisNormal from '@ohif/core/src/utils/calculateScanAxisNormal';
```

Comment:
You can't access src directory from @ohif/core - I know it mostly builds, but not consistently.  You will have to export it from the utils module in @ohif/core and then import it from @ohif/core and then extract from utils.